src/Server.php

```diff
@@ -77,7 +101,7 @@ class Server extends AbstractServer
      * @return Server
      * @throws Exception\InvalidArgumentException If function invalid or not callable.
      */
-    public function addFunction($function, $namespace = '')
+    public function addFunction($function, $namespace = ''): Server
     {
         if (! is_callable($function)) {
```

Comment:
I don't follow. What should be dropped?

Reply:
@Xerkus thought we can add param type hint `callable` and drop the if statement here, but we can't because it comes from parent.

I am suggesting updating laminas-server before, but this is a lot of job as well...

Reply:
Agreed. These are changes that I would like to see.

Can we complete this PR, then update `laminas-server` and afterwards update usage in `laminas-json-server`?

Reply:
I made a note about the update of `laminas-server` and thus will resolve this conversation.